Userland/Libraries/LibWeb/Layout/FrameBox.cpp

```diff
@@ -24,6 +24,10 @@ FrameBox::~FrameBox()
 
 void FrameBox::prepare_for_replaced_layout()
 {
+    if (dom_node().nested_browsing_context() == nullptr) {
+        return;
+    }
```

Comment:
This is fine, but please drop the curly braces for if statements with a single line body. Same below.